std/uuid.d

```diff
@@ -193,7 +197,9 @@ public struct UUID
             ///Version 4 (Random)
             randomNumberBased = 4,
             ///Version 5 (Name based + SHA-1)
-            nameBasedSHA1 = 5
+            nameBasedSHA1 = 5,
+            ///Version 7 (milliseconds since unix epoch + random)
+            timestampRandom = 7
```

Comment:
what happened to version 6?

Reply:
v8 is not better version of v7, v8 is just another approach